validate.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var badgeMinimum int
+
+const inadequateBadge = "badges/inadequate.svg"
+const adequateBadge = "badges/adequate.svg"
+
+var badgeRex = regexp.MustCompile("(?i)(\\[!\\[[a-zA-Z0-9_ ]*\\]\\([0-9a-z.\\/:?=-]*\\)\\]\\([0-9a-z.\\/:-]*\\))")
+
+func checkBadges(username string, reponame string) (string, error) {
+	resp, err := http.Get(fmt.Sprintf("https://raw.githubusercontent.com/%s/%s/master/README.md", username, reponame))
```

Comment:
Yep, where default is master. I think you can just shove it into this URL at that point.